src/main/java/com/epam/api/services/PropertyService.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.api.services;
+
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class PropertyService {
```

Comment:
it is not a service

Reply:
why?

Reply:
It is class for reading properties, nothing deals with service

Reply:
+